packages/bpk-component-modal/src/BpkModalV2/BpkModal.tsx

```diff
@@ -120,7 +120,7 @@ export const BpkModalV2 = (props: Props) => {
         dialog.addEventListener('click', (event: Event) => {
           const { target } = event;
 
-          if (target instanceof HTMLElement && target.id === `${id}`) {
+          if ((target as HTMLElement) && target === dialog) {
```

Comment:
what is the reason for this change? I guess the left side will always be true as it's just casting?

Reply:
Oops, that was not suppose to be there

Reply:
Sorry, I did not pushed all my changes. I will comment on what the changes are doing.